index.html

```diff
@@ -36,5 +36,190 @@
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </span>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <span class="tech-stack">
+            <ul>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+          </span>
+        <div class="button">
+          <button>see projects</button>
+        </div>
+      </div>
+      </div>
+  
+    <div class="cards">
+      
+      <div class="screenshoot">
+        <img src="images/Snapshoot Portfolio (3).png" alt="portifolio screenshoot">
+      </div>
+
+      <div class="description">
+        <h2>Multi-Post Stories</h2>
+        <span class="part">
+          <ul>
+            <li id="canopy">CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </span>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <span class="tech-stack">
+          <ul x>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+        </span></div>
+        <div class="button">
+        <button>see projects</button></div>
+        
+      </div>
+    
+    <div class="cards">
+      
+      <div class="screenshoot">
+        <img src="images/Snapshoot Portfolio (4).png" alt="portifolio screenshoot">
+      </div>
+
+      <div class="description">
+        <h1>Tonic</h1>
+        <span class="part">
+          <ul>
+            <li id="canopy">CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </span>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <span class="tech-stack">
+          <ul x>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+        </span></div>
+        <div class="button">
+        <button>see projects</button>
+        </div>
+      </div>
+    
+    <div class="cards">
+      
+      <div class="screenshoot">
+        <img src="images/Snapshoot Portfolio (5).png" alt="portifolio screenshoot">
+      </div>
+
+      <div class="description">
+        <h2>Multi-Post Stories</h2>
+        <span class="part">
+          <ul>
+            <li id="canopy">CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </span>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <span class="tech-stack">
+          <ul x>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+        </span></div>
+        <div>
+        <button>see projects</button>
+        </div>
+        
+      </div>
+    </div>
+    </section>
+    <section class="about me">
```

Comment:
- [ ] As demanded by project requirements, make sure the about section is laid out using CSS flexbox.
![image](https://github.com/Afsheen96/Portfolio/assets/94551484/ae882829-67f3-4796-9f75-9da5d5cc2992)